src/components/pages/Categories.js

```diff
@@ -0,0 +1,7 @@
+function Categories() {
+  return (
+    <h1>Under construction</h1>
+  );
```

Comment:
- [ ] [REQUIRED] Kindly display a button reading "Check Status" in the categories component as stated in the requirements.